src/neofs_testlib/env/templates/neofs_env_config.yaml

```diff
@@ -0,0 +1,50 @@
+binaries:
+    neofs_cli:
+        repo: 'nspcc-dev/neofs-node'
+        version: 'v0.39.2'
```

Comment:
Can be 0.40.1 now.

Reply:
It was all tested with 0.39.2. I will update versions to current ones in a separate change during tests sync with actual node version.